packages/grafana-ui/src/components/Table/styles.ts

```diff
@@ -48,10 +49,10 @@ export function useTableStyles(theme: GrafanaTheme2, cellHeightOption: TableCell
 
       '&:hover': {
         overflow: overflowOnHover ? 'visible' : undefined,
-        width: overflowOnHover ? 'auto' : undefined,
-        height: overflowOnHover ? 'auto' : `${rowHeight - 1}px`,
+        width: stringShouldWrap ? ' auto' : 'auto !important',
```

Comment:
maybe there is here an extra space: ' auto'

Reply:
Fixed!